### DISCUSSION TITLE
Раздел Почта и уведомления дополнить отправкой в Telegram через бота

### DISCUSSION THREAD

**Discussion by 112358aalexx** [mikopbx/Core#54] [Ideas (Идеи)]
Предлагаю добавить отдельную вкладку, где описать возможность использования telegram-ботов типа @gmailBot транслирующих письма из ящика в телеграм.

**Comment by boffart** [comment]
Тема интересная. 
Во многих местах напрашивается введение "**Канала оповещений**"
К примеру стоим модуль "**Telegram**"  добавляется соответствующий канал. 
Добавляем модуль "**b24**" аналогично добавляется канал для оповещений. 

В планах есть, но не ближайших.
